peer_heap.js

```diff
@@ -121,8 +120,9 @@ PeerHeap.prototype.clear = function clear() {
 PeerHeap.prototype.add = function add(peer) {
     var self = this;
 
-    var score = peer.scoreStrategy.getScore();
-    var i = self.push(peer, score);
+    var range = peer.pendingWeightedRange();
+
+    var i = self.push(peer, range[1]);
```

Comment:
isn't this the end? I thought we wanted to have the start of range be the heap key?